services-gateway-netty/src/main/java/io/scalecube/services/gateway/ws/GatewayMessages.java

```diff
@@ -1,9 +1,7 @@
 package io.scalecube.services.gateway.ws;
 
 import io.scalecube.services.api.ServiceMessage;
-import io.scalecube.services.api.ServiceMessage.Builder;
 import io.scalecube.services.exceptions.DefaultErrorMapper;
-import java.util.Optional;
 
 final class GatewayMessages {
```

Comment:
Plz make this class public and make it's static methods public (yes, you would have to write some javadoc, unfortunately). It was a mistake to make it package-private in this first place.

Reply:
Oh, it's a trap =)